docs/fastdds/dds_layer/core/policy/eprosimaExtensions.rst

```diff
@@ -202,6 +202,36 @@ XML
     :start-after: <!-->PUBSUB_API_CONF_PUBSUB_DISABLE_POSITIVE_ACKS
     :end-before: <!--><-->
 
+
+.. _flowcontrollersqos:
+
+FlowControllersQos
+^^^^^^^^^^^^^^^^^^
+
+This QoS configures the list of flow controllers of a participant, so they can later be used on
+its DataWriters.
+It consists of a vector of |FlowControllerDescriptor-api|, which has the following fields.
```

Comment:
```suggestion
It consists in a vector of |FlowControllerDescriptor-api|, which has the following fields.
```

Reply:
I think we can use `It is a vector of ...`